x/mint/testutil/expected_keepers_mocks.go

```diff
@@ -67,6 +67,22 @@ func (mr *MockStakingKeeperMockRecorder) StakingTokenSupply(ctx interface{}) *go
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "StakingTokenSupply", reflect.TypeOf((*MockStakingKeeper)(nil).StakingTokenSupply), ctx)
 }
 
+// StakingTokenSupply mocks base method.
+func (m *MockStakingKeeper) TotalBondedTokens(ctx context.Context) (math.Int, error) {
+	m.ctrl.T.Helper()
+	ret := m.ctrl.Call(m, "StakingTokenSupply", ctx)
```

Comment:
I think we have to call "TotalBondedTokens" instead of "StakingTokenSupply"?

Reply:
good catch ty